powersimdata/data_access/scenario_list.py

```diff
@@ -75,7 +75,7 @@ def add_entry(self, scenario_info):
         scenario_info.move_to_end("id", last=False)
         table.reset_index(inplace=True)
         entry = pd.DataFrame({k: [v] for k, v in scenario_info.items()})
-        table = table.append(entry)
+        table = pd.concat([table, entry])
```

Comment:
What's the purpose of this change?

Reply:
When I was testing the functionality, there was a deprecation warning indicating that `append` will be deprecated in a future release of `pandas` and `concat` should be used instead

Reply:
Makes sense, thanks for being proactive on this.